### ISSUE TITLE
Problem running the demo project

### ISSUE THREAD

**Issue by Isuru-Nanayakkara** [shin8484/PopupWindow#13]
I get the error **No such module 'PopupWindow'** when I try to run the demo project on Xcode 9.3.

**Comment by Isuru-Nanayakkara** [comment]
And it's missing a lot of files.

**Comment by kyramckenna** [comment]
Yes the sample app pod file contains different files to the ones you get when you run pod install
